web/src/containers/ApdPreviousActivityTable.js

```diff
@@ -41,7 +41,7 @@ const ApdPreviousActivityTable = ({
           </th>
         </tr>
         <tr>
-          <th aria-hidden="true" />
+          <td className="th" aria-hidden="true" />
```

Comment:
ahahahaha *sob*

Reply:
I am 100% open to a different class name. 😬